addons/website_blog/controllers/main.py

```diff
@@ -191,7 +231,7 @@ def blog_feed(self, blog, limit='15', **kwargs):
     @http.route([
             '''/blog/<model("blog.blog", "[('website_id', 'in', (False, current_website_id))]"):blog>/post/<model("blog.post", "[('blog_id','=',blog[0])]"):blog_post>''',
     ], type='http', auth="public", website=True)
-    def blog_post(self, blog, blog_post, tag_id=None, page=1, enable_editor=None, **post):
+    def blog_post(self, blog, blog_post, tag_id=None, page=1, enable_editor=None, is_transitioning=False, **post):
```

Comment:
dont add 'useless' (params that will not change logic from controller) param, you could get it from post if really needed.

But in your case just dont add it in values, but use request.params.get('transition') in your template if you really want it... And imo, variable is too long for a visible params.